pipelines/Jenkinsfile

```diff
@@ -0,0 +1,104 @@
+import static groovy.json.JsonOutput.toJson
+import static groovy.json.JsonOutput.prettyPrint
```

Comment:
I'm using some dependencies that will require approval in Jenkins administration. For testing, to ensure my upload and eviction logic was correct, I ran https://github.com/jenkinsci/jenkinsfile-runner with `--no-sandbox` enabled. Definitely not something you'd do in prod but I'm using it basically as a REPL to ensure I'm writing good code.